result/metrics.yaml

```diff
@@ -71,7 +71,7 @@ components:
           type: array
           items:
             $ref: './lag.yaml#/components/schemas/Lag.Metric'
-        lacp_lag_member_metrics:
+        lacp_metrics:
           type: array
           items:
             $ref: './lacplagmember.yaml#/components/schemas/LacpLagMember.Metric'
```

Comment:
As I mentioned in the PR description, only the name is changed, not the underlying object name. If we do that, it requires more changes in the implementation. Anyway, it can be done - including renaming the source file name.